torchgeo/datasets/eurosat.py

```diff
@@ -318,3 +318,25 @@ class EuroSAT100(EuroSAT):
         'val': '3e3f8b3c344182b8d126c4cc88f3f215',
         'test': 'f908f151b950f270ad18e61153579794',
     }
+
+
+class SpatialSplitEuroSAT(EuroSAT):
+    """A subclass of the EuroSAT dataset class overriding the default splits.
```

Comment:
The fact that it's a subclass, and the parent class, will be automatically documented. Instead, let's talk about what it is and why it's useful.